webapp/api/store.py

```diff
@@ -75,6 +76,8 @@ def __init__(self, store=None, testing=False, cache=True):
             self.session = api.requests.Session()
         else:
             self.session = api.requests.CachedSession(timeout=(1, 6))
+            talisker.requests.configure(self.session)
```

Comment:
Unless there's a particular reason, I would configure the session regardless of testing or caching - it should affect neither of those things?